src/blurredLocation.js

```diff
@@ -10,6 +10,14 @@ BlurredLocation = function BlurredLocation(options) {
 
   options.gridSystem = options.gridSystem || require('./core/gridSystem.js');
   options.Interface = options.Interface || require('./ui/Interface.js');
+  options.onDrag = options.onDrag || function onDrag() {
```

Comment:
so our default onDrag should be this function:

```js
function onDrag(event) {
  function changeVal(result) {
    $("#location").val(result.results[0].formatted_address);
  }
  getPlacenameFromCoordinates(getLat(), getLon(), changeVal)
}
```

Also, since this is interface code, it belongs in the interface class!